src/Client/GRPC/BaseClient.php

```diff
@@ -19,6 +19,11 @@
 
 abstract class BaseClient implements ServiceClientInterface
 {
+    const RETRYABLE_ERRORS = [
+        StatusCode::RESOURCE_EXHAUSTED,
+        StatusCode::UNAVAILABLE,
+        StatusCode::UNKNOWN,
+    ];
```

Comment:
We ought to expand this to include the ones here https://github.com/temporalio/sdk-core/blob/81ece42e3a4fda1b019d9a5d8d263916f73726a6/client/src/retry.rs#LL22C1-L22C1

Reply:
We'll also want to double check that the retry logic itself is the same - using exponential backoff etc. The logic is described later in the file here: https://github.com/temporalio/sdk-core/blob/81ece42e3a4fda1b019d9a5d8d263916f73726a6/client/src/retry.rs#L182

DM me if you have any questions, thanks!